roles/common/tasks/rbac_setup.yml

```diff
@@ -7,6 +7,8 @@
     oauth_password: "{{ oauth_superuser_client_password }}"
     ldap_user: "{{ mds_super_user }}"
     ldap_password: "{{ mds_super_user_password }}"
+    client_cert: "{{ kafka_broker_cert_path }}"
```

Comment:
rename this to make it different from uri module's arguments
maybe mtls_client_cert